tests/scheduler/test_baseline.py

```diff
@@ -11,12 +11,52 @@
 from rubin_scheduler.scheduler.example import example_scheduler
 from rubin_scheduler.scheduler.model_observatory import ModelObservatory
 from rubin_scheduler.scheduler.schedulers import CoreScheduler
-from rubin_scheduler.scheduler.surveys import BlobSurvey, GreedySurvey, generate_dd_surveys
+from rubin_scheduler.scheduler.surveys import (
+    BlobSurvey,
+    GreedySurvey,
+    ScriptedSurvey,
+    generate_ddf_scheduled_obs,
+)
 from rubin_scheduler.scheduler.utils import SkyAreaGenerator, calc_norm_factor_array
 
 SAMPLE_BIG_DATA_FILE = os.path.join(get_data_dir(), "scheduler/dust_maps/dust_nside_32.npz")
 
 
+class ModelObservatoryWindy(ModelObservatory):
```

Comment:
This seems like something that would be better handled by customizing ModelObservatory rather than a whole new subclass?

Reply:
I did it as a subclass because I don't think there's a normal use case where we would want a model observatory where it's always windy. Didn't want to add a kwarg that could only ever be used to break things in production.